app/views/settings/admission_webhook/index.html.slim

```diff
@@ -0,0 +1,65 @@
+= render 'settings/apply'
+
+.alert.alert-warning role='alert'
+  p Warning: Entering invalid values for any of the following settings will cause
+    the nodes to enter into a broken state.
+
+h2 Admission Webhook
+
+p
+  | Enable Kubernetes ValidatingAdmissionWebhook admission controller. Once enabled
```

Comment:
you need to add a space at $ or at the beginning of the next line, otherwise it reads `Once enabledit is possible to...`

Reply:
Thanks!